e2e/test/DeviceClientX509AuthenticationE2ETests.cs

```diff
@@ -140,20 +149,22 @@ private async Task SendMessageTest(ITransportSettings transportSetting)
             }
         }
 
+        private ITransportSettings CreateHttpTransportSettingWithCertificateRevocationCheck()
+        {
+            TlsVersions.Instance.CertificateRevocationCheck = true;
```

Comment:
This can be a problem for tests that run in parallel, because changing this static instance will impact all the others. You should make sure tests that change this value do not run in parallel.

Reply:
Even if all of these tests set value to "true", is a problem running it in parallel?

Reply:
Nope, as soon as one of them wants to be false, it will be though.

Reply:
Is it good idea to user ordering? Or is there a better way to specify "not to run tests in parallel"?